operations/paymentrequests.md

```diff
@@ -0,0 +1,264 @@
+| `State` | [PaymentRequestState](#payment-request-state) | required | A payment request state. |
+| `Amount` | int | required | Amount of the payment request. |
+| `Type` | [PaymentRequestType](#payment-request-type) | required | A payment request type. |
+| `Reason` | [PaymentRequestReason](#payment-request-reason) | required | A payment request reason. |
+| `ExpirationUtc` | string | required | Date and time of the payment request's expiration in ISO 8601 format. |
+| `Description` | string | required | Description of the payment request. |
+| `Notes` | string | optional | Payment request's notes. |
+
+#### Payment request state
+
+* `Pending` - payment request is active and waiting for completion.
+* `Completed` - payment request was fulfilled.
+* `Canceled` - payment request was canceled.
+* `Expired` - payment request is past its expiration date and no longer active.
+
+#### Payment request type
+
+* `Payment` - indicates that a [Payment](payments.md) is requested.
+* `Preauthorization` - indicates that a [Preauthorization](preauthorizations.md) is requested.
+
+#### Payment request reason
+
+* `PaymentCardMissing` - payment card is missing.
+* `PaymentCardDeclined` - payment card was declined.
+* `Prepayment` - prepayment is requested.
+* `Fee` - fee is requested.
+* `Other` - other payment request reason.
+
+## Add payment requests
+
+Creates a payment request to the specified [Customer](customers.md#customer).
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/paymentRequests/add`
+
+```javascript
+{
+    "Client": "Sample Client 1.0.0",
+    "AccessToken": "DEF9B0AA35F74220ADC3AFA900D0EB24-251F227C3344EEEF232B01038D6B780",
+    "ClientToken": "3640AA564052470681BFAFA900D0EA09-C6BE8CB1B41BE4A8F90E7BA1ADBEB54",
+    "PaymentRequests": [
+        {
+            "AccountId": "8466dfdd-0964-4002-8719-afa900d0f1ba",
+            "Amount": {
+                "Currency": "EUR",
+                "Value": 10
+            },
+            "Type": "Payment",
+            "Reason": "PaymentCardMissing",
+            "ExpirationUtc": "2023-02-20T12:00:00.000Z",
+            "Description": "Payment required",
+            "Notes": "Internal notes."
+        }
+    ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `PaymentRequests` | array of [Payment request parameters](#payment-request-parameters) | required | Payment requests to be added. |
+
+#### Payment request parameters
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `AccountId` | string | required | Unique identifier of the [Customer](customers.md#customer) to which the payment request is issued. |

```

Comment:
```suggestion
| `AccountId` | string | required | Unique identifier of the [Customer](customers.md#customer) account to which the payment request is issued. |
```
As before, I just prefer if the terminology in the description ties up with the name of the property.